lib/wasix/src/lib.rs

```diff
@@ -169,6 +169,20 @@ pub enum SpawnError {
     /// Some other unhandled error. If you see this, it's probably a bug.
     #[error("unknown error found")]
     UnknownError,
+    #[error("runtime error")]
+    Runtime(#[from] WasiRuntimeError),
+    #[error("{0}")]
+    Other(String),
```

Comment:
Rather than adding a `Other(String)` variant, would it be better to modify `UnknownError` to be `UnknownError(#[cause] Box<dyn std::error::Error + Send + Sync>)`?

Reply:
Yeah, though the primary motivation here is for `anyhow::Error`, so we'll a std error wrapper for anyhow. 

We should have a more extensive custom error type for `Runtime::load_module` though.